libraries/shared/storage/backfill/storage_value_loader_test.go

```diff
@@ -69,8 +79,14 @@ var _ = Describe("StorageValueLoader", func() {
 			Repository:        &mocks.MockStorageRepository{},
 		}.NewTransformer
 
-		initializers = []storage.TransformerInitializer{initializerOne, initializerTwo}
-		runner = backfill.NewStorageValueLoader(bc, nil, initializers, blockOne, blockTwo)
+		initializerThree = storage.Transformer{
```

Comment:
just curious - why did we needed a 3rd block in this test scenario?

Reply:
we probably don't 😅 

think I misread the pattern here as having a new block for each address, but on a second glance that's not the case. Will remove ✂️